packages/dds/tree/src/simple-tree/arrayNode.ts

```diff
@@ -690,6 +556,101 @@ abstract class CustomArrayNodeBase<const T extends ImplicitAllowedTypes> extends
 	public get length() {
 		return fail("Proxy should intercept length");
 	}
+
+	// We manually add [Symbol.iterator] to the dispatch map rather than use '[fn.name] = fn' as
```

Comment:
This comment is outdated now - it can be removed, the method is very self-explanatory now.